geo-types/src/geometry_collection.rs

```diff
@@ -23,9 +23,9 @@ use std::ops::{Index, IndexMut};
 /// ```
 /// use std::convert::TryFrom;
 /// use geo_types::{Point, point, Geometry, GeometryCollection};
-/// let p = point!(x: 1.0, y: 1.0);
+/// let p = point! { x: 1.0, y: 1.0 };
```

Comment:
Hrmm, why macro delimiter bikeshedding?

Reply:
Ok, I'm willing concede that curly braces are now the *one true way* we should write these constructor macros.

I guess they're kind of evocative of "direct struct field" syntax, which is arguably useful. 👍